source/gloperate-text/source/stages/GlyphRenderStage.cpp

```diff
@@ -48,7 +48,7 @@ void GlyphRenderStage::onProcess(gloperate::AbstractGLContext *)
 
     if (!fbo)
     {
-        fbo = globjects::Framebuffer::defaultFBO();
+        fbo = globjects::Framebuffer::defaultFBO().get();

```

Comment:
As i understand it, `targetFramebuffer.value()` does not transfer ownership while `defaultFBO` does. 
Is this the way to go then? 

    auto fbo = targetFramebuffer.value();

    std::unique_ptr<globjects::Framebuffer> resource_wrapper;
    if (!fbo)
    {
        resource_wrapper = globjects::Framebuffer::defaultFBO();
        fbo = resource_wrapper.get();
    }

Or should the resource wrapper better be a member (`m_fbo`) that is lazy initialized?